formatting_Utils.py

```diff
@@ -0,0 +1,25 @@
+from decimal import *
```

Comment:
please rename the file to 
formating_utils.py
;)

Reply:
Done